src/System.Data.SqlClient/tests/ManualTests/README.md

```diff
@@ -1,6 +1,32 @@
 # SqlClient Manual Tests
 
-These tests require dedicated test servers, so they're designed to be run manually using a custom set of connection strings. These connection strings should be added as environment variables: TEST\_NP\_CONN\_STR & TEST\_TCP\_CONN\_STR. TEST\_NP\_CONN\_STR is a named pipes connection string to the test server, and TEST\_TCP\_CONN\_STR is a TCP connection string. Each protocol can be specified in the Server name parameter. These tests also assume the sample database "Northwind" exists in the target server. This sample database can be found [here](https://msdn.microsoft.com/en-us/library/mt710790.aspx).
+These tests require dedicated test servers, so they're designed to be run manually using a custom set of connection strings. 
 
-Instructions for running tests: [Unix](https://github.com/dotnet/corefx/blob/master/Documentation/building/cross-platform-testing.md) and [Windows](https://github.com/dotnet/corefx/blob/master/Documentation/building/windows-instructions.md).
-Documentation for connection string parameters: [SqlConnection.ConnectionString](https://msdn.microsoft.com/en-us/library/system.data.sqlclient.sqlconnection.connectionstring.aspx).
\ No newline at end of file
+## Prerequisites
+
+- CoreFX building. you need to be able to do a successful `build -allconfigurations` and run the standard tests, [Unix](https://github.com/dotnet/corefx/blob/master/Documentation/building/cross-platform-testing.md) or [Windows](https://github.com/dotnet/corefx/blob/master/Documentation/building/windows-instructions.md)
```

Comment:
I would suggest to use `build.cmd` and `build.sh`, as they should suffice on Windows and Linux respectively, as you only need platform specific binaries to test.
AFAIK `build -allconfigurations` doesn't work on unix platforms, it only works on windows and it takes way longer as it generates `nuget` packages for all the namespaces, which we really don't need for running manual tests.

Reply:
The corefx guide tells you to use allconfigurations and in my experience doing that is needed to force all the dependencies to restore and package generation, and yes it does take a long time. If you're going to do the EFCore tests later though you do need the nuget packages to be generated. for those instructions the user should follow the corefx guides I linked to. 

This is really telling them to get the whole repo building first. Shall I take out the explicit command and just refer them to the links?

Reply:
My recommendation since this README is only for manual tests, keep the `build.cmd` and `build.sh` and add a note, if intention is to test with `EFCore` later, then build with `build -allconfigurations` instead.

Reply:
@Wraith2 Kindly add the following to the README:
Use `build.cmd` for windows and `build.sh` for Linux to build CoreFX. 
N.B. if you want to run the EFCore tests later you will need to `build -allconfigurations` to generate the NuGet packages, `build -allconfigurations` works only on windows.